Lombiq.Hosting.Tenants.Maintenance/StaggeredMigrations.cs

```diff
@@ -0,0 +1,85 @@
+using Lombiq.Hosting.Tenants.Maintenance.Constants;
+using Lombiq.Hosting.Tenants.Maintenance.Models;
+using OrchardCore.ContentFields.Fields;
+using OrchardCore.ContentFields.Settings;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.Data.Migration;
+using System.Threading.Tasks;
+
+namespace Lombiq.Hosting.Tenants.Maintenance;
+
+public sealed class StaggeredMigrations : DataMigration
+{
+    private readonly IContentDefinitionManager _contentDefinitionManager;
+
+    public StaggeredMigrations(IContentDefinitionManager contentDefinitionManager) =>
+        _contentDefinitionManager = contentDefinitionManager;
+
+    public async Task<int> CreateAsync()
+    {
+        await _contentDefinitionManager.AlterPartDefinitionAsync(nameof(StaggeredMaintenancePart), part => part
+            .WithField<NumericField>(nameof(StaggeredMaintenancePart.ProcessingStep), field => field
+                .WithDisplayName("Processing Step")
+                .WithSettings(new NumericFieldSettings
+                {
+                    Hint = "The number of tenants to be processed in each step or in case of parallel processing the" +
+                           " number of tenants started at once.",
+                }))
+            .WithField<TimeField>(nameof(StaggeredMaintenancePart.TimeSpanBetweenBatches), field => field
+                .WithDisplayName("Time Span Between Batches")
+                .WithSettings(new TimeFieldSettings
+                {
+                    // 1 second.
+                    Step = "1",
+                    Hint = "The time span between batches of tenants to be processed in hh:mm:ss format. " +
+                           "If StaggeredMaintenanceOptions is set, this value will be ignored.",
+                }))
+            .WithField<BooleanField>(nameof(StaggeredMaintenancePart.RunParallel), field => field
+                .WithDisplayName("Run Parallel")
```

Comment:
```suggestion
                .WithDisplayName("Run In Parallel")
```
BTW currently (while it's set to "Run Parallel") it's displayed as "RunParallel".